modules/nixos.nix

```
@@ -165,5 +164,42 @@ in {
           concatStringsSep " " ruleString
       ) (filter (f: f.enable && f.source != null) (attrValues files));
     }) (filterAttrs (_: u: u.files != {}) config.homes);
+    systemd.services = mapAttrs' (name: {files, ...}: {
+      name = "hjem-monitor-" + name;
+      value = {
+        enable = true;
+        wants = ["systemd-tmpfiles-setup.service" "nix-daemon.socket"];
+        after = ["nix-daemon.socket"];
+        wantedBy = ["default.target"];
```

```suggestion
        wantedBy = ["multi-user.target"];
```

`multi-user.target` is a dependency of `default.target`, which runs slightly before `default.target`.